brim/add-scripts.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -euo pipefail
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+
+declare -a REPOS=(
+    'https://github.com/salesforce/ja3.git'
+    'https://github.com/salesforce/hassh.git'
+)
+
+for REPO in "${REPOS[@]}"; do
+    TMPDIR=$(mktemp -d)
+    ZSCRIPTS_DIR="$SCRIPT_DIR/../scripts/site"
+    SCRIPTNAME="${REPO/\.git/}"
+    SCRIPTNAME="${SCRIPTNAME/*\//}"
+    git clone "$REPO" "$TMPDIR"
+    for DIR in zeek bro scripts; do
+        if [ -d "$TMPDIR/$DIR" ]; then
+            mv "$TMPDIR/$DIR" "$ZSCRIPTS_DIR/$SCRIPTNAME"
```

Comment:
I suggest that this `rm -rf "$ZSCRIPTS_DIR/$SCRIPTNAME"` before the move, and that it write a file that records $REPO and the result of `git describe --tags --dirty` so that we record the git commit/version.